From ed9260d04f86ac2f697fc21124663d6975086268 Mon Sep 17 00:00:00 2001
From: Arnaud Quette <arnaud.quette@free.fr>
Date: Mon, 11 Apr 2016 09:29:27 +0200
Subject: [PATCH] nut-scanner: additional search path

Prepend "/usr/" to the provided --libdir, in case this one points at "/lib/..."
or similar, and not to "/usr/lib/..."
---
 tools/nut-scanner/nutscan-init.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/nut-scanner/nutscan-init.c b/tools/nut-scanner/nutscan-init.c
index 5f08ea2..5537cd6 100644
--- a/tools/nut-scanner/nutscan-init.c
+++ b/tools/nut-scanner/nutscan-init.c
@@ -45,6 +45,7 @@ int nutscan_load_upsclient_library(const char *libname_path);
 /* FIXME: would be good to get more from /etc/ld.so.conf[.d] */
 char * search_paths[] = {
 	LIBDIR,
+	"/usr"LIBDIR,
 	"/usr/lib64",
 	"/lib64",
 	"/usr/lib",
-- 
2.1.4

